src/cloudant/design_document.py

```diff
@@ -49,6 +49,45 @@ def __init__(self, database, document_id=None):
             self.setdefault(prop, dict())
 
     @property
+    def st_indexes(self):
+        """
+        Provides an accessor property to the Cloudant geospatial
+        (a.k.a. Cloudant Geo) indexes dictionary in the locally cached
+        DesignDocument.  Each Cloudant Geo index is a JSON object within the
+        ``st_indexes`` containing an index name and a javascript function.
+
+        Note: To make it easier to work with Cloudant Geo documents, it is best
+        practice to create a separate design document specifically for
+        Cloudant Geo indexes.
+
+        Geospatial index example:
+
+        .. code-block:: python
+
+            # Add the Cloudant Geo index to ``st_indexes`` and save the design document
+            ddoc = DesignDocument(self.db, '_design/ddoc001')
+            ddoc['st_indexes'] = {
+                'geoidx': {
+                    'index': 'function(doc) { '
+                             'if (doc.geometry && doc.geometry.coordinates) { '
+                             'st_index(doc.geometry);}} '
+                }
+            }
+            ddoc.save()
+
+        Once the Cloudant Geo index is saved to the remote database, you can
+        query the index with a GET request to the ``_geo`` endpoint.
+        Example query: ``/$DATABASE/_design/doc/_geo/geoidx?<query-parameters>``
```

Comment:
I appreciate that you added the test after my last comment which would let people fine how to do this, I do think the doc should help guide them to what is necessary.
Perhaps we could just link to http://python-cloudant.readthedocs.io/en/latest/getting_started.html#endpoint-access to help them understand that to do the query they will need to form the request themselves.